compiler/one-cmds/onelib/CfgRunner.py

```diff
@@ -46,6 +46,14 @@ def __init__(self, path):
         self.import_drivers = [
             'one-import-bcq', 'one-import-onnx', 'one-import-tf', 'one-import-tflite'
         ]
+        # parse group option
+        GROUP_OPTION_KEY = 'include'
+        if self.cfgparser.has_option('onecc', GROUP_OPTION_KEY):
+            groups = self.cfgparser['onecc'][GROUP_OPTION_KEY].split()
+            for o in groups:
+                assert (o.startswith('O'))
```

Comment:
It would be better to raise an exception to print error message, e.g., "Group option name must start with O".